Assets/Scripts/Terrain/StreamingWorld.cs

```diff
@@ -772,10 +774,10 @@ private IEnumerator UpdateLocation(int index, bool allowYield)
                     }
 
                     // Perform layout and yield after each block is placed
-                    ContentReader contentReader = DaggerfallUnity.Instance.ContentReader;
-                    for (int y = 0; y < height; y++)
+                    int blockIndex = 0;
+                    for (int y = 0; y < height; ++y)
```

Comment:
Why change the for loop to pre-increment y? I don't think it makes any difference to how the loop functions, but it's strange and seems an odd thing to change here. Is there a reason I am missing?

Reply:
It's a minor style change, the main change there is to increment blockIndex on each step instead of calculating it every time. And I always use pre-increment in loops since it is slightly more efficient and recommended. But I can revert it to post-increment to remain consistent with the rest of the DFU codebase if that's the coding convention, no problem on my side.